volatility3/framework/plugins/windows/callbacks.py

```
@@ -107,19 +366,22 @@ def list_notify_routines(
             try:
                 symbol_offset = ntkrnlmp.get_symbol(symbol_name).address
             except exceptions.SymbolError:
-                vollog.debug(f"Cannot find {symbol_name}")
+                # vollog.debug(f"Cannot find {symbol_name}")
```

Was this just for testing or do you think the message has no value?  Happy to entertain the idea, I genuinely don't know how useful it is, but I prefer knowing when things have gone wrong than just silently skipping over them.  Perhaps moving if from `debug` to `constants.LOGLEVEL_VVVV` would be better?